git_machete/exceptions.py

```diff
@@ -12,3 +12,10 @@ def __str__(self) -> str:
 class StopTraversal(Exception):
     def __init__(self) -> None:
         pass
+
+
+class UnprocessableEntityHTTPError(Exception):
+    """This exception is raised when Github API returns HTTP status code 422 - Unprocessable Entity.
+    Such a situation occurs when trying to do something not allowed by github, egz. assign as a reviewer someone from outside organization or create a pull request which is already created.
+    """
+    pass
```

Comment:
`def __init__` ? not sure if there's any difference for exception classes whether the c'tor is present or not

Reply:
(either way, pls keep consistent across all exception classes)

Reply:
__init__ is redundant here, because the whole logic is inherited from Exception. For the consistency, let's pass message to c'tor explicitly.